README.md

```diff
@@ -230,7 +234,7 @@ Entra domains and subscriptions.
    - A secret with the ID from the subscription that everything should be deployed into.
    - A secret with the ID of the App Registration created previously.
 6. Create a copy of one of the environments under the [operations](./operations) folder.
-   1. Name the copy off of the name of the new environment.
+   1. Name the copy off of the name of the new environment. ` Can we give an example here?`
```

Comment:
Can we give an example of how the copy would be named? I think it might helpful.

Reply:
For example, `internal`.